pom.xml

```diff
@@ -68,6 +68,93 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <tychoVersion>2.6.0</tychoVersion>
   </properties>
+  <dependencyManagement>
```

Comment:
not sure I fully understand plus many of these are really old.  Error prone for example is on 2.11.0 currently (although that one is broken for libraries like mockito) so 2.10.0 is appropriate one.  Guava version here is vulnerable last I checked, it needs to be the current one.  Closure compiler is 6 years old.  JSR305 will do nothing but cause issues if it is even used.  It was replaced entirely by spotbugs annotations.  Not clear overall why this level of changes are needed.  I've never ran into any issue and we actively run this on 100s of repos from jdk 6 to jdk 17 without any issue.